plugins/modules/identity/ipa/ipa_sudorule.py

```diff
@@ -361,6 +385,7 @@ def ensure(module, client):
 def main():
     argument_spec = ipa_argument_spec()
     argument_spec.update(cmd=dict(type='list', elements='str'),
+                         cmdgroup=dict(type='list', elements='str'),
```

Comment:
You need to add this option to the documentation.

Reply:
Yes, sorry, added the example but forgot the documentation part.